packages/core/src/tracing/sentrySpan.ts

```diff
@@ -401,6 +402,8 @@ export class SentrySpan implements SpanInterface {
       childSpan.spanRecorder.add(childSpan);
     }
 
+    addChildSpanToSpan(this, childSpan);
```

Comment:
```suggestion
    // For now, we capture this here as well in order to ensure backwards compatibility
    // eventually this will only be captured in `startSpan` methods
    addChildSpanToSpan(this, childSpan);
```